src/bjointsp/main.py

```diff
@@ -19,8 +19,8 @@
 
 
 # solve with heuristic; interface to place-emu: triggers placement
-def place(network_file, template_file, source_file, fixed_file=None, prev_embedding_file=None, cpu=None, mem=None,
-          dr=None, networkx=None, write_result=True):
+def place(network_file, template_file, source_file, source_template_object=False, fixed_file=None,
```

Comment:
Can you add a comment here what `source_template_object` is for?

Reply:
Sure